posts/when_to_fold_em.md

```diff
@@ -0,0 +1,48 @@
+# When to fold 'em
+
+Some years ago, I worked as part of a development team on a project to upgrade a tangled legacy system managing a company’s payment systems. Two months in, and we had very little to show for it - pages of diagrams that looked like spiderwebs, a few outages caused by failed attempts to untangle the pile, and a rising level of frustration.
+
+That company had spent thousands of pounds, not to mention missed out on a bunch of other opportunities - and not got a good return on that investment. Stakeholders weren’t any happier than we were, and it was time to decide what to do next.
+
+> We’ve learned some things about what we’re trying to tackle here. Let’s give it another couple of months and see if we can make any progress.
+
+Of course, a month later, nothing had changed - it was the same team, with the same tools, and the same level of support.
+
+> We’ve already sunk so much money into this. We can’t throw that away now. Let’s keep fighting!
+
+This is the **sunk cost** fallacy in action - treating what you’ve already lost as a factor in the decision on what to do next.
+
+Our brains were compiled in caves, and some of its software is very out-of-date. Today, we’re going to do some bug-fixing.
+
+
+## Let the project die - kill it, if you have to
+
+But what is the bug? Aren’t you right to care about wasting money?
+
+Sure, waste is bad. But if you’re looking at in the past, it’s already wasted. Since the past loss is happening no matter what, *only look at the future results* if you want to make the right decision.
```

Comment:
looking *at* in the past? should "at" be in this sentence?

Reply:
This was actually supposed to be 
> at it in the past
with the "it" being the waste. I think that reads slightly better to me than
> at the past